downloader/src/main/kotlin/vcs/GitRepo.kt

```diff
@@ -58,6 +58,25 @@ class GitRepo : GitBase() {
                     return super.getInfo().copy(path = manifestFile.relativeTo(workingDir).invariantSeparatorsPath)
                 }
 
+                override fun getNested(): Map<String, VcsInfo> {
+                    val paths = runRepoCommand(workingDir, "list", "-p").stdout.lines().filter { it.isNotBlank() }
```

Comment:
Again, is the filter really needed, or just added for safety even if you never saw blank lines?

Reply:
The last line is always blank, but it seemed safer to me to check all lines.